distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc

```diff
@@ -0,0 +1,344 @@
+
+.[[_isr_attributes_table]]ISR: Attributes in this group relate to intelligence, search, and reconnaissance.
+[cols="1,2,1,1,1" options="header"]
+|===
+|Term
+|Definition
+|Datatype
+|Constraints
+|Example Value
+
+|isr.frequency-hertz
+|A frequency observation, typically of radio waves.
+|Float
+|normalized to Hz
+|
+
+|isr.target-id
+|A target identifier.
+|String
+|< 1024 characters
+|
+
+|isr.target-category-code
+|A STANAG 3596 target category.
+|String
+|One of the values enumerated in STANAG 3596
+|
+
+|isr.platform-id
+|A platform identifier.
+|String
+|< 1024 characters
+|
+
+|isr.original-source
+|A STANAG 4545 ISOURCE.
+|String
+|ISOURCE per STANAG 4545
+|
+
+|isr.organizational-unit
+|An organizational unit.
+|String
+|< 1024 characters
+|
+
+|isr.niirs
+|The quality or detail level of an image. (National Imagery Interpretability Rating Scale)
+|Integer
+|1-9
+|
+
+|isr.nato-reporting-code
+|A reporting code as defined in STANAG 4545.
+|String
+|One of the values in STANAG 4545
+|
+
+|isr.mission-id
+|A mission identifier.
+|String
+|< 1024 characters
+|
+
+|isr.sensor-type
+|A STANAG 4545 sensor type.
+|STANAG 4545 ACFTB TRE
+|String
+|
+
+|isr.sensor-id
+|A STANAG 4545 sensor identifier.
+|String
+|STANAG 4545 ACFTB TRE
+|
+
+|isr.cloud-cover
+|An imagery cloud cover percentage.
+|Integer
+|0-100
+|
+
+|isr.category
+|A STANAG 4559 image or video category.
+|String
+|
+|
+
+|isr.image-id
+|An ISR imagery identifier.
+|
```

Comment:
Is there a reason why these don't have a type? We may want to make a note if there is a reason.